Sources/NIO/ByteBuffer-core.swift

```diff
@@ -49,6 +49,13 @@ extension _ByteBufferSlice {
     }
 }
 
+extension _ByteBufferSlice: CustomStringConvertible {
+    @usableFromInline
+    var description: String {
+        return "\(lowerBound)..<\(upperBound)"
```

Comment:
I followed the ByteBuffer: CustomStringConvertible documentation here, I assumed it includes "slice: 256..<768" because _ByteBufferSlice is to be considered an implementation detail. Should I modify both?

Reply:
`slice` is a property name, whereas this is a type name. I don’t think it matters too much to add it, especially as this should now be meaningful standalone.